app/models/form.rb

```diff
@@ -172,7 +172,7 @@ def questions(reload = false)
   end
 
   def visible_questionings
-    questionings.reject{|q| q.hidden?}
+    questionings.reject{ |q| q.hidden? || q.multimedia? }
```

Comment:
It might, but my linter doesn't catch that at the moment. Still tweaking its options. I can change it.